docs/quickstart.md

```diff
@@ -0,0 +1,177 @@
+# Quickstart with firecracker-containerd
+
+This quickstart guide provides simple steps to get a working
+firecracker-containerd environment, with each of the major components built from
+source.  Once you have completed this quickstart, you should be able to run and
+develop firecracker-containerd (the components in this repository), the
+Firecracker VMM, and containerd.
+
+This quickstart will clone repositories under your `$HOME` directory and install
+files into `/usr/local/bin`.
+
+1. Get an AWS account
+2. Launch an i3.metal instance running Amazon Linux 2 (you can find it in the
```

Comment:
Would it be worth expanding the instructions around "launch an i3.metal". I'm thinking something like "Here is a link to aws-cli getting started documentation. Here is a list of aws commands you can run to startup an instance" and include teardown instructions at the end.